Vungle/src/main/java/com/mopub/mobileads/VungleRewardedVideo.java

```diff
@@ -271,7 +279,12 @@ public void onAdStart(@NonNull String placementReferenceId) {
                 MoPubLog.log(getAdNetworkId(), CUSTOM, ADAPTER_NAME, "onAdStart - Placement ID: " +
                         placementReferenceId);
                 mIsPlaying = true;
-                MoPubRewardedVideoManager.onRewardedVideoStarted(VungleRewardedVideo.class, mPlacementId);
+
+                if (mInteractionListener != null) {
+                    mInteractionListener.onAdShown();
+                    mInteractionListener.onAdImpression();
```

Comment:
across Rewarded, Interstitial and Banners we would have to come back and check if onAdStart is better for putting onAdImpression , or when sending onAdAvailabilityUpdate as true